### DISCUSSION TITLE
Cannot connect to server: Timeout...

### DISCUSSION THREAD

**Discussion by morawer** [reflex-dev/reflex#2275] [Q&A] [answered]
Hello everyone,
I have created and hosting (in two different places) a little static web with Reflex. This web has a form in the first page to take some datas from the user and when you push the submit button make a request to API in Miro to create a sticker.

When i go to the web in 5 or 10 second appear this message and the web doesn't respond.

The problem is this:
![Captura desde 2023-12-07 18-20-25](https://github.com/reflex-dev/reflex/assets/54649242/43548c36-1c71-4d89-93bb-f0574b2af73b)

Do you know how i can fix this issue? I have been searching info about this but i don't find the key.

Thank you in advance, Sorry if it is a easy issue.

**Comment by picklelo** [comment]
Does your app use state? If so, you will also have to host your backend and set the `api_url`. See here: https://reflex.dev/docs/hosting/self-hosting/

**Reply by morawer** [reply]
Hello @picklelo, thank you for your answer.
Yes, i manage a state in the project.
Sincerely i dont get to resolve this issue, it's my first web deploy and the problem continue same.
If you allow me i would like to show you any pieces of my project...

This is my rxconfig.py:
![Captura desde 2023-12-09 12-00-40](https://github.com/reflex-dev/reflex/assets/54649242/f76b964e-7085-4aa2-9333-80b3eeb5f5a1)

This is the tree of the project:
![Captura desde 2023-12-09 11-39-40](https://github.com/reflex-dev/reflex/assets/54649242/96cc5d12-9373-4c1f-bb2f-a16ac7e405fc)

And into the folder ./incidencias_web i have the python files.

The error message is the same:
![Captura desde 2023-12-09 11-44-11](https://github.com/reflex-dev/reflex/assets/54649242/4bee909d-6800-4927-a397-ec11a5844399)

I have been trying the docs says but i can´t get to link the public with the backend. Could you get me any light about this?

Thank you in advance.

**Reply by picklelo** [reply, marked as answer]
@morawer You need to host your backend separately, Vercel won't do it automatically. If you join our Discord, and post in the #deployment channel I can give you access to our hosting service which should make deploying much easier.

**Reply by morawer** [reply]
Thank you @picklelo i made the deploy. If i can get to solve the issue i will come back here to post the solution.

**Reply by morawer** [reply]
Problem solved @picklelo,the web works good. Thanks for all!
